### DISCUSSION TITLE
Figma Exporting and managing styles locally?

### DISCUSSION THREAD

**Discussion by oilsinwater** [dicebear/dicebear#161] [Q&A] [answered]
Hi I am really loving Dicebear, thank you for creating it!

Question: what i the best way to I export my work from Figma and then manage my own style locally using the CLI integration and VS Code? 

I've been trying to figure out a workflow, but I'm stumped on how to do that.

**Comment by FlorianKoerner** [comment, marked as answer]
Hello Philliph. Two weeks ago we released a Figma Exporter plugin to make exporting easier.

You can find the plugin here:
https://www.figma.com/community/plugin/1005765655729342787/DiceBear-Exporter

Unfortunately, the documentation is still a work in progress. But maybe our compatible Figma files will help you to understand how it works:
https://github.com/dicebear/dicebear/blob/main/SOURCES.md

The file for Croodles describes in comments how it works:
https://www.figma.com/file/ZHPv3t2L6Asuuql9ND4q1L/croodles?node-id=7%3A393

If you need help, feel free to let me know. I'll try to finish the documentation in a timely manner.

**Reply by oilsinwater** [reply]
Thank you! Those examples were extremely helpful! Giving it a try, and seems to be working out!

**Comment by FlorianKoerner** [comment]
If you want to make your avatar style available under the MIT license, I will be happy to help you prepare your Figma file for export. Just let me know.

**Comment by oilsinwater** [comment]
Wonderful, appreciate the offer and will give it some thought! Thank you again!!!!
